WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Carriers and Rates/ShippingLabelCarriers.swift

```diff
@@ -50,7 +48,10 @@ struct ShippingLabelCarriers: View {
                                        description: Localization.emptyStateDescription,
                                        image: .productErrorImage)
                         }
-                        .frame(width: geometry.size.width, height: geometry.size.height, alignment: .center)
+                        .frame(width: geometry.size.width,
+                               alignment: .center)
+                        .position(x: geometry.size.width/2,
+                                  y: geometry.size.height/2)
```

Comment:
I still can see a small problem when viewing the empty-screen in accessibility fonts or in a small device screen.

See how the content on top is inaccessible due to the scroll inset.



https://user-images.githubusercontent.com/562080/120340763-ec0ed500-c2bb-11eb-9c03-75c391a7040d.mov